cpp/ql/src/Critical/FileClosed.qll

```diff
@@ -8,10 +9,19 @@ predicate closed(Expr e) {
   )
 }
 
+/** An expression for which there exists a function call that might close it. */
 class ClosedExpr extends PointsToExpr {
   ClosedExpr() { closed(this) }
 
   override predicate interesting() { closed(this) }
 }
 
+/**
+ * Holds if `fc` is a call to function that opens a file which might be closed. For example:
```

Comment:
```suggestion
 * Holds if `fc` is a call to a function that opens a file that might be closed. For example:
```

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which

Reply:
"which" vs. "that" truly is my grammar kryptonite. I've fixed it now.